pyproject.toml

```diff
@@ -33,6 +33,7 @@ dependencies = [
     'mctx>=0.0.5',
     'networkx>=3.3',
     'pytest>=6.2.1',
+    "numpyro>=0.19.0",
```

Comment:
this isn't actually used in `evolve_trials`, but it is in `aif_likelihood`, which we'll fix in a separate PR